zeus/utils/testing.py

```diff
@@ -159,3 +162,48 @@ def end_window(
         self.logger.info("Measurement window '%s' ended (%s).", key, measurement)
 
         return measurement
+
+
+NUM_CPUS = 2
+
+
+class MOCKCPU(CPU):
+    """Control a single MOCK CPU for testing."""
+
+    def __init__(self, index):
+        """Initialize the MOCKCPU with a specified index for testing."""
+        self.index = index
+        self.cpu_energy = itertools.count(start=1000, step=10)
+        self.dram_energy = (
+            itertools.count(start=200, step=5) if self.index % 2 == 0 else None
+        )
```

Comment:
These hardcoded values (1000, 10, 200, 5) are directly asserted in monitor tests. It doesn't make sense for them to be inside the main `zeus` package then, because changing these values to something else will end up requiring the modification of two different files. Plus, `NUM_CPUS` is also a testing parameter, but because you put all this inside `zeus`, you had to import it inside the testing code which is weird. I would suggest to move all the Mock CPUs to `tests`.

Reply:
I think the reason why I put it in utils was because I had to import it into t`est_monitory.py` because I couldn't do `zeus.tests.utils....`. Is there a way to import other modules within the test directory?

Reply:
I would just make `test_monitor.py` longer. Those aren't reused elsewhere anyway AFAIK